pandas/tests/io/parser/conftest.py

```diff
@@ -53,11 +53,11 @@ def csv_dir_path(datapath):
 
 
 @pytest.fixture
-def csv1(csv_dir_path):
```

Comment:
What's the reason for doing this instead of editing `csv_dir_path`?

Reply:
csv_dir_path is used across quite a few tests. We would have to move all csvs from io/parser/data to io/data/csv if we wanted to do this.

Reply:
Gotcha. I think makes sense to do that but can be done as a follow up